### DISCUSSION TITLE
3번째 멘토링 시간 변경

### DISCUSSION THREAD

**Discussion by hi-hyein** [hi-hyein/starCraftUnitOOP#17] [General]
@jongfeel 
멘토님 안녕하세요!
어머니가 다음 주 수술을 하게 되어서
주말동안 입원을 하게되었어요
그래서 요번주 주말 프로젝트 진행과 다음 주 멘토링 진행은 어려울 것 같아 한 주 미루고자합니다!
7월1일 목요일 멘토님 스케줄 괜찮으시면 변경부탁드리겠습니다~!

(아카데미 컨퍼런스 일정은 문제없어요!)

**Comment by jongfeel** [comment]
엇 그렇군요.

아카데미 컨퍼런스도 끝나자 마자 바로 병원 가셔야 겠네요.
26일 토요일 시간 조정 필요하시면 얘기해 주세요.

스케줄 조정은 바로 하겠습니다.

**Reply by hi-hyein** [reply]
감사합니다!
컨퍼런스 일정에 영향은 없어요!ㅎㅎ
